headers/data/overlay10.h

```diff
@@ -93,6 +93,7 @@ extern int16_t WONDER_CHEST_EXP_BOOST;
 extern int16_t SPAWN_CAP_WITH_MONSTER_HOUSE;
 extern int16_t POISON_DAMAGE_COOLDOWN;
 extern int16_t LEECH_SEED_DAMAGE_COOLDOWN;
+extern int16_t ITEM_HIT_CHANCE;
```

Comment:
Can we name this `THROWN_ITEM_HIT_CHANCE`? Just "item" seems too generic.

Reply:
:+1: